pkg/gitreceive/build.go

```diff
@@ -84,7 +84,7 @@ func build(
 	}
 	defer func() {
 		if err := os.RemoveAll(tmpDir); err != nil {
-			fmt.Errorf("unable to remove tmpdir %s (%s)", tmpDir, err)
+			log.Info("unable to remove tmpdir %s (%s)", tmpDir, err)
```

Comment:
`govet` caught this. `fmt.Errorf` does nothing here, so I decided to replace this with a log so at least the error is now logged.